multicluster/apis/multicluster/v1alpha1/tunnelendpoint_types.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2022 Antrea Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// TunnelEndpointSpec defines the desired state of TunnelEndpoint
+type TunnelEndpointSpec struct {
+	// Active shows the Gateway Node is active or not.
+	Active bool `json:"active,omitempty"`
+	// ClusterID of the member cluster.
+	ClusterID string `json:"clusterID,omitempty"`
+	// Name of Gateway Node
+	Name string `json:"name,omitempty"`
+	// Subnets used by the member cluster including PodCIDRs and Service ClusterIP range.
+	Subnets []string `json:"subnets,omitempty"`
```

Comment:
From openflow perspective, there is no difference between PodCIDRs and ServiceCIDRs, so I just use one field to store all. maybe I can split them in case there is any potential case I haven't seen.

Reply:
done, refined TunnelEndpoint spec with new PodCIDRs and ServiceCIDR.